gc/base/GCExtensionsBase.hpp

```diff
@@ -250,7 +250,7 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 	MM_SublistPool rememberedSet;
 #endif /* OMR_GC_MODRON_SCAVENGER */
 #if defined(OMR_GC_STACCATO)
-	MM_RememberedSetWorkPackets* staccatoRememberedSet; /**< The Staccato remembered set used for the write barrier */
+	MM_RememberedSetWorkPackets* sATBBarrierRememberedSet; /**< The snapshot at the beginning barrier remembered set used for the write barrier */
```

Comment:
To make this change self contained (not dependent  related OpenJ9 changes), you should keep both staccatoRememberedSet and sATBBarrierRememberedSet. Once this is merged, and after OpenJ9 is merged, you will have another small cleanup OMR change to remove staccatoRememberedSet.

Reply:
same applies for other name changing including methods and classes - all should have temporary duplicates

Reply:
added staccatoRememberedSet back untill openj9 changes are in and we can remove it in the next set of changes.